automated-release/artifacts/mappings/mapping1.txt

```diff
@@ -0,0 +1 @@
+mapping 1
```

Comment:
yeah just a random file with random content, it's meant to emulate the `crashlytics` output mappings we bundle as artifacts in other projects